webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/usermanagement/index/RoleIndex.java

```diff
@@ -7,11 +7,12 @@
  */
 package io.camunda.webapps.schema.descriptors.usermanagement.index;
 
+import io.camunda.webapps.schema.descriptors.backup.Prio5Backup;
 import io.camunda.webapps.schema.descriptors.usermanagement.UserManagementIndexDescriptor;
 import io.camunda.webapps.schema.entities.usermanagement.EntityJoinRelation.EntityJoinRelationFactory;
 import io.camunda.webapps.schema.entities.usermanagement.EntityJoinRelation.IdentityJoinRelationshipType;
 
-public class RoleIndex extends UserManagementIndexDescriptor {
+public class RoleIndex extends UserManagementIndexDescriptor implements Prio5Backup {
```

Comment:
Removed by mistake today in main in [commit](https://github.com/camunda/camunda/commit/23404a94f23f85263cfe5b2a7df18f7d1ec521a6#diff-b1529f330014dfe26200f492ceae14fc9c3999d18607958db41fa2d2e6356ca1R13)